assets/post_install_windows.bat

```diff
@@ -13,4 +13,4 @@ echo Pinning BLAS implementation to OpenBLAS.
 echo libblas=*=*openblas >> "%PREFIX%\conda-meta\pinned"
 
 echo Running mne sys_info.
-"%PREFIX%\Scripts\conda" run mne sys_info
+"%PREFIX%\Scripts\conda" run mne sys_info || echo
```

Comment:
Turns out (I think!) that `mne sys_Info` was the problematic line, so I'll effectively `|| true` it like we do in the macOS script for example